src/lib/__tests__/mergeSchemas.js

```diff
@@ -91,3 +92,53 @@ describe("convection link", () => {
     })
   })
 })
+
+describe("stiched schema regressions", () => {
+  it.only("union in interface fragment issue", async () => {
+    const artworkResponse = {
+      id: "banksy-di-faced-tenner-21",
+      sale_ids: ["foo-foo"],
+    }
+
+    const sales = [
+      {
+        id: "foo-foo",
+        _id: "123",
+        currency: "$",
+        is_auction: true,
+        increment_strategy: "default",
+      },
+    ]
+
+    const result = await runQueryMerged(
+      `
+      {
+        artwork(id:"banksy-di-faced-tenner-21") {
+          id
+          context {
+            ... on Node {
+              __id
+              __typename
+            }
+          }
+        }
+      }
+    `,
+      {
+        artworkLoader: async () => artworkResponse,
+        salesLoader: async () => sales,
+        relatedFairsLoader: async () => ({}),
```

Comment:
I haven’t been using the `async` keyword in anger yet; this is equal to `() => Promise.resolve({})` ?

Reply:
Yes, exactly.